src/components/molecules/RadioField/RadioField.tsx

```diff
@@ -138,9 +138,12 @@ const Input = styled.input<InputStatus & GroupingControlsProps>`
     `}
 `;
 
-export interface RadioField extends FieldProps, InputProps, GroupingControlsProps {}
+export interface RadioField extends FieldProps, InputProps, GroupingControlsProps {
```

Comment:
Apparently the coverage has gone down based on this - we might want to look at whether a test is needed here.

Reply:
https://github.com/zopaUK/react-components/pull/648/checks?check_run_id=1649671866

Reply:
I meant it's not required to merged but we should test the new logic 90% of the times